Chart.yaml

```diff
@@ -1,21 +1,6 @@
 apiVersion: v2
-name: cloudbolt-collector-helm
-description: A Helm chart for Kubernetes
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+name: cloudbolt-collector
+description: Deploys an application that extracts Kubernetes cluster utilization data and sends it to the CloudBolt platform.
 type: application
-# This is the chart version. This version number should be incremented each time you make changes
-# to the chart and its templates, including the app version.
-# Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: v0.20.0
```

Comment:
Since we made a change to the values.yml, should this increment? `appVersion` hasn't, but technically the chart has, though it's noop.